packages/spektate/src/repository/IAzureDevOpsRepo.ts

```diff
@@ -52,13 +52,13 @@ export const getPullRequest = (
           id: pr.pullRequestId,
           mergedBy: pr.closedBy
             ? {
-                imageUrl: pr.closedBy._links?.avatar?.href
-                  ? pr.closedBy._links?.avatar?.href
-                  : pr.closedBy.imageUrl,
-                name: pr.closedBy.displayName,
-                url: pr.url,
-                username: pr.closedBy.uniqueName
-              }
+              imageUrl: pr.closedBy._links?.avatar?.href
```

Comment:
Since you're in the area let's add negative test cases 
<img width="701" alt="image" src="https://user-images.githubusercontent.com/3119513/79785028-0c6a1300-82f8-11ea-9eab-9b2759acacbc.png">